waspc/examples/todoApp/todoApp.wasp

```diff
@@ -109,5 +112,9 @@ action toggleAllTasks {
 }
 
 job mySpecialJob {
-  perform: import { foo } from "@ext/jobs/bar.js"
+  executor: PgBoss,
+  perform: {
+    fn: import { foo } from "@ext/jobs/bar.js",
+    options: {=json retryLimit: 1 json=}
```

Comment:
These are default options (`pg-boss` specific, here) that are included in all invocations of the `perform` `fn`, so I scoped it as such. We may have other options higher up, like for `pg-boss` workers perhaps.

Reply:
Sounds good!
One thought -> instead of JSON, you could instead teach Wasp what are the PG-boss options, by constructing a piece of AppSpec that describes them. This means though that we have to duplicate knowledge about those options, and update it if pg-boss changes, but it also means Wasp can check in compile time if they make sense.
Once we add union types, we can have these options be different for different executors.

Reply:
Yeah that is true. I did consider that, but was worried about how many flags we would need, and you are right about the maintenance burden of `pg-boss` version changes. If we were going to act on them in any way, I think that would be the way to go, but since we just shuttle the data I was thinking JSON may be ideal here. But good to keep options in mind!